Content.Shared/GameObjects/EntitySystemMessages/WeightlessChangeMessage.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.GameObjects.EntitySystemMessages
+{
+    [Serializable, NetSerializable]
+    public sealed class WeightlessChangeMessage : EntitySystemMessage
+    {
+        public readonly IEntity Entity;
+        public readonly bool IsNowWeightless;
```

Comment:
Could just call it like "Weightless" but this is pedantic so you can ignore it if you want.